### PR TITLE
Update stacked timeseries and 3d scatter plots

### PR DESCRIPTION
**Pull request by Bobfrat**
@DanielJMaher @oceanzus 
Please review. Added some pep8 formatting as well

### CODE REVIEW

**Comment by Bobfrat** [comment]
screen shot 2016-12-13 at 2 33 00 pm

**Comment by Bobfrat** [comment]
Addresses Redmine tickets [10913](https://uframe-cm.ooi.rutgers.edu/issues/10913) and [10765](https://uframe-cm.ooi.rutgers.edu/issues/10765)

**Comment by oceanzus** [comment]
Looks good from here.

File: ooiservices/app/uframe/plot_tools.py [hunk 1]
```diff
@@ -685,16 +700,25 @@ def plot_3d_scatter(self, fig, ax, x, y, z, title='', xlabel='', ylabel='', zlab
             axis_font = axis_font_default
 
         cmap = plt.cm.jet
```
Comment (DanielJMaher) [inline ooiservices/app/uframe/plot_tools.py]: They've asked us to use Red-Blue Divergent as the colormap for this.  Should just be plt.cm.RdBu

**Comment by DanielJMaher** [comment]
Besides my inline comment, I have other feedback.

Gliders have problems plotting this data type, so if they attempt to plot a glider with this we should gracefully tell them that they can't.

PD0 is the bin_depth variable.  If that is not present in a Non-Glider ADCP then there is an issue within uFrame.  We should throw a graceful error message saying that the variable is not present and to contact the data team.  All of these plots should be against bin depth.

Let's discuss what is rational to do.

**Pull request merged**